documents/final-submission-document/chapters/4_implementation.tex

```diff
@@ -4,12 +4,31 @@ \chapter{Implementation and Testing} % Realisierung und Test
 
 
 \section{Development Setup}
-% - e2e tests
-% - angular cli
-% - build script / ci
-% - docker-compose, CA automated with scripts etc. SE stuff
 
-\section{Encountered Problems}
+Figure \fullref{fig:development-setup} illustrates the development setup in the form of an UML deployment diagram.
+A developer connects via his browser to the reverse proxy that serves the XMPP-Grid Broker web application.
```

Comment:
I think using "reverse proxy" here can be a bit confusing, as it actually is a web server. I suggest changing it to that, do you agree?

Reply:
We leave it that way (as discussed) :crying_cat_face: :disappointed:

Reply:
![](https://media1.tenor.com/images/6db91599aa40fa4d258d865615b4fe09/tenor.gif?itemid=7870358)